modin/pandas/test/test_io.py

```diff
@@ -899,13 +935,17 @@ def test_read_csv_default_to_pandas_url(self, request):
     @pytest.mark.parametrize("nrows", [21, 5, None])
     @pytest.mark.parametrize("skiprows", [4, 1, 500, None])
     def test_read_csv_newlines_in_quotes(self, nrows, skiprows):
+        if Backend.get() == "Omnisci" and skiprows in [4, 500] and nrows is None:
```

Comment:
`skiprows in [4, 500]` feels wrong - we'd have to always keep in sync `.parametrize()` above and this `.xfail()`.
```suggestion
        if Backend.get() == "Omnisci" and skiprows not [0, 1, None] and nrows is None:
```
I suggest listing supported values for `skiprows` instead